src/Cache/DefaultCacheFactory.php

```diff
@@ -122,7 +122,7 @@ public function buildQueryCache(EntityManagerInterface $em, string|null $regionN
             $em,
             $this->getRegion(
                 [
-                    'region' => $regionName ?: Cache::DEFAULT_QUERY_REGION_NAME,
+                    'region' => $regionName ?? Cache::DEFAULT_QUERY_REGION_NAME,
```

Comment:
Passing an empty string would get us the default region previously. I would bet that someone relies on that. 🙈